### ISSUE TITLE
Sensu Go handler seems to crash when solving a request handling an Azure OIDC token

### ISSUE THREAD

**Issue by bobbsickle** [sensu/sensu-go#3514]
Currently trying to use Azure OIDC to login to a docker hosted sensu go instance. However, when it seems like I have passed the authorization step I get a blank page from sensu. Looking at the docker logs I got the following:

```json
{"component":"etcd","level":"info","msg":"http: panic serving 172.17.0.1:45076: runtime error: invalid memory address or nil pointer dereference\ngoroutine 95820 [running]:\nnet/http.(*conn).serve.func1(0xc000ed2be0)\n\t/usr/local/go/src/net/http/server.go:1767 +0x139\npanic(0x1528f60, 0x26f6950)\n\t/usr/local/go/src/runtime/panic.go:679 +0x1b2\ngithub.com/sensu/sensu-enterprise-go/backend/apid/routers.extractClaims(0xc0010af9f0, 0xc000cd78e5, 0x5, 0xc000cd78d6, 0x6, 0xc0010af9f0, 0x0, 0x0, 0xc000ace900, 0x0, ...)\n\t/Users/nikkixdev/go/src/github.com/sensu/sensu-enterprise-go/backend/apid/routers/oidc.go:464 +0x122\ngithub.com/sensu/sensu-enterprise-go/backend/apid/routers.(*OIDCRouter).callback(0xc000ad2230, 0x7f1ab1a26fd0, 0xc000c37890, 0xc000129300)\n\t/Users/nikkixdev/go/src/github.com/sensu/sensu-enterprise-go/backend/apid/routers/oidc.go:298 +0x56f\nnet/http.HandlerFunc.ServeHTTP(0xc000ad2400, 0x7f1ab1a26fd0, 0xc000c37890, 0xc000129300)\n\t/usr/local/go/src/net/http/server.go:2007 +0x44\ngithub.com/sensu/sensu-go/backend/apid/middlewares.LimitRequest.Then.func1(0x7f1ab1a26fd0, 0xc000c37890, 0xc000129300)\n\t/Users/nikkixdev/go/pkg/mod/github.com/sensu/sensu-go@v0.0.0-20191217234800-bef1e698e674/backend/apid/middlewares/limitrequest.go:26 +0x217\nnet/http.HandlerFunc.ServeHTTP(0xc0011e4940, 0x7f1ab1a26fd0, 0xc000c37890, 0xc000129300)\n\t/usr/local/go/src/net/http/server.go:2007 +0x44\ngithub.com/sensu/sensu-go/backend/apid/middlewares.SimpleLogger.Then.func1(0x1ba6580, 0xc001231260, 0xc000129300)\n\t/Users/nikkixdev/go/pkg/mod/github.com/sensu/sensu-go@v0.0.0-20191217234800-bef1e698e674/backend/apid/middlewares/logging.go:25 +0x1ff\nnet/http.HandlerFunc.ServeHTTP(0xc0011e4960, 0x1ba6580, 0xc001231260, 0xc000129300)\n\t/usr/local/go/src/net/http/server.go:2007 +0x44\ngithub.com/gorilla/mux.(*Router).ServeHTTP(0xc000108c00, 0x1ba6580, 0xc001231260, 0xc000ac9100)\n\t/Users/nikkixdev/go/pkg/mod/github.com/gorilla/mux@v1.7.1/mux.go:212 +0xe2\nnet/http.serverHandler.ServeHTTP(0xc000d6c460, 0x1ba6580, 0xc001231260, 0xc000ac9100)\n\t/usr/local/go/src/net/http/server.go:2802 +0xa4\nnet/http.(*conn).serve(0xc000ed2be0, 0x1bad9c0, 0xc001d0bb40)\n\t/usr/local/go/src/net/http/server.go:1890 +0x875\ncreated by net/http.(*Server).Serve\n\t/usr/local/go/src/net/http/server.go:2927 +0x38e\n","pkg":"","time":"2020-01-22T11:26:30Z"}
```
 It seems as if the handler is crashing when trying to formulate a response. Any idea if i am doing something wrong or could this be an issue outside of my control?

Additionally, here are some of my configs incase they come in handy:

```json
{
   "type": "oidc",
   "api_version": "authentication/v2",
   "metadata": {
      "name": "NAME"
   },
   "spec": {
      "additional_scopes": [
         "email"
      ],
      "client_id": "CLIENT_ID",
      "client_secret": "CLIENT_SECRET",
      "redirect_uri": "http://localhost:8080/api/enterprise/authentication/v2/oidc/callback",
      "server": "https://login.microsoftonline.com/TENNENT_ID/v2.0",
      "groups_claim": "groups",
      "groups_prefix": "oidc:",
      "username_claim": "email",
      "username_prefix": "oidc:"
   }
}
```

Thanks

**Comment by echlebek** [comment]
Thanks for filing this issue and stack trace, we'll look into it!

**Comment by echlebek** [comment]
The bug is in the private sensu-enterprise-go repo, we'll fix it internally and respond to this issue once the build ships.

**Comment by palourde** [comment]
This was actually fixed in 5.18.1 via https://github.com/sensu/sensu-enterprise-go/pull/887
